### DISCUSSION TITLE
How to enable menu keyboard navigation

### DISCUSSION THREAD

**Discussion by JonathanBaq** [sibiraj-s/ngx-editor#505] [Q&A] [answered]
Hi, 

I wanted to make the menu keyboard navigable for accessibility. So far I added 'tabIndex=0' to menu items but I noticed that while I can navigate by tabbing now, I can't select/toggle the menu items by pressing 'space' or 'enter'. Interestingly enough, pressing works when I have an assistive device like VoiceOver on Mac turned on with ctrl+option+space but I want this to work even without an assistive device. 
I remedied this by calling the editor toggle commands manually on 'keydown' but I noticed that not all menu items have toggle commands which leaves me a bit stuck. 

Any suggestions are welcome, thank you!

**Comment by sibiraj-s** [comment, marked as answer]
Several improvements to accessibility were made, This should be working now.
